packages/runtime/src/runtime.ts

```diff
@@ -422,6 +423,22 @@ export class Runtime {
 		this.store.accounts.set(feeSink.address, feeSink);
 	}
 
+	/**
+	 * Set up account from deployer to runtime
```

Comment:
I am not sure what we are doing here. Could you explain?

Reply:
In runtime, I do not see we have method that adding account to the store so I create this method so as to help me adding account from the config to the store